README.md

```diff
@@ -18,37 +18,49 @@
+
+Then, install every package in the set:
 
 ```
 $ spago ls packages | cut -f 1 -d ' ' | xargs spago install
 ```
 
-to install every package in the set. Before deploying an updated package set, someone should check that the memory required to hold the package set's externs files does not exceed that of the try.purescript.org server.
+If this command fails because a package that previously existed in the package set has been removed, then edit the `spago.dhall` file to remove the offending packages (you can also just remove all dependencies). Then, re-run the installation command above.
+
+Before deploying an updated package set, someone (your reviewer) should check that the memory required to hold the package set's externs files does not exceed that of the try.purescript.org server.
 
 ### Control Features via the Query String
 
 Most of these features can be controlled not only from the toolbar, but also using the [query parameters](https://en.wikipedia.org/wiki/Query_string):
 
 - **Load From Gist**: Load PureScript code from Gist id using the `gist` parameter
-    - Example: `gist=37c3c97f47a43f20c548` will load the code from this Gist if the file was named `Main.purs`
+
+  - Example: `gist=37c3c97f47a43f20c548` will load the code from this Gist if the file was named `Main.purs`
 
 - **View Mode**: Control the view mode using the `view` parameter
-    - Options are: `code`, `output`, `both` (default)
-    - Example: `view=output` will only display the output
+
+  - Options are: `code`, `output`, `both` (default)
+  - Example: `view=output` will only display the output
 
 - **Auto Compile**: Automatic compilation can be turned off using the `compile` parameter
-    - Options are: `true` (default), `false`
-    - Example: `compile=false` will turn auto compilation off
+
+  - Options are: `true` (default), `false`
+  - Example: `compile=false` will turn auto compilation off
 
 - **JavaScript Code Generation**: Print the resulting JavaScript code in the output window instead of the output of the program using the `js` parameter
-    - Options are: `true`, `false` (default)
-    - Example: `js=true` will print JavaScript code instead of the program's output
+
+  - Options are: `true`, `false` (default)
+  - Example: `js=true` will print JavaScript code instead of the program's output
 
 - **Session**: Load code from a session which is stored with [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) using the `session` parameter
-    - Usually managed by Try PureScript
-    - Example: `session=9162f098-070f-4053-60ea-eba47021450d` (Note: will probably not work for you)
-    - When used with the `gist` query parameter the code will be loaded from the Gist and not the session
+  - Usually managed by Try PureScript
+  - Example: `session=9162f098-070f-4053-60ea-eba47021450d` (Note: will probably not work for you)
+  - When used with the `gist` query parameter the code will be loaded from the Gist and not the session
```

Comment:
I'm sorry about the autoformatting. I can revert it if desired and there are no changes to this part of the documentation.

Reply:
Yes please (sorry for the bother)